core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/signer/AwsV4aHttpSigner.java

```diff
@@ -31,15 +32,15 @@
 @SdkPublicApi
 public interface AwsV4aHttpSigner extends AwsV4FamilyHttpSigner, HttpSigner<AwsCredentialsIdentity> {
     /**
-     * The AWS region name to be used for computing the signature. This property is required.
-     * TODO(sra-identity-and-auth): Should this be a list or rename to SIGNING_SCOPE?
+     * The AWS region-set to be used for computing the signature. This property is required.
      */
-    SignerProperty<String> REGION_NAME = SignerProperty.create(String.class, "SigningScope");
+    SignerProperty<Set<RegionScope>> REGION_SET = SignerProperty.create("RegionSet");
```

Comment:
I think RegionScope is already supposed to be a set-ish thing. Right now we only support one value (no commas), but that could change in the future as the signers evolve.

It's also weird to model RegionScope here explicitly, instead of just using String, like we do with region.

Should probably just do `SignerProperty<Set<String>>`? CRT literally just takes a `String`, but I don't understand scope enough to know if that's a good idea or not....

Reply:
FWIU, we wanted stronger validation than String, so we use `RegionScope`. We also couldn't tell if `RegionScope` was supposed to represent a set, since it disallows commas.

Reply:
I'm personally inclined to use a simple string, and the implementation can do validation upon signing, but it's up to the team. I believe the argument for not doing this is that potential errors in the input can be detected when adding the region-set value, instead of when signing happens.

Reply:
Concluded offline that we will re-use RegionScope in a more limited fashion, `RegionSet`, and use it instead.